test/acceptance/features/bindAppToService.feature

```diff
@@ -217,3 +217,96 @@ Feature: Bind an application to a service
         And jq ".status.conditions[] | select(.type=="InjectionReady").status" of Service Binding Request "binding-request-empty-app" should be changed to "False"
         And jq ".status.conditions[] | select(.type=="InjectionReady").reason" of Service Binding Request "binding-request-empty-app" should be changed to "EmptyApplicationSelector"
 
+    Scenario: Service Binding with Strimzi Kafka Cluster database without applicationSelector
+        Given Imported Nodejs application "nodejs-empty-app" is running
+        * Strimzi Kafka operator is installed
+        * Kafka cluster "strimzi-kafka-cluster" is running
+        When Service Binding Request is applied to connect the database and the application
+            """
+            apiVersion: apps.openshift.io/v1alpha1
+            kind: ServiceBindingRequest
+            metadata:
+                name: binding-request-strimzi-kafka-no-app
+            spec:
+                backingServiceSelector:
+                    group: kafka.strimzi.io
+                    kind: Kafka
+                    resourceRef: strimzi-kafka-cluster
+                    version: v1beta1
+                    id: kaf
+                    customEnvVar:
```

Comment:
since there is no application, does `customEnvVar` really make any difference?

Reply:
removing strimzi kafka case